dashbuilder-backend/dashbuilder-dataset-sql/src/main/java/org/dashbuilder/dataprovider/sql/JDBCUtils.java

```diff
@@ -253,25 +254,9 @@ public static ColumnType calculateType(int sqlDataType) {
                 return ColumnType.DATE;
             }
 
-            /*case Types.BINARY:
-            case Types.VARBINARY:
-            case Types.LONGVARBINARY:
-            case Types.NULL:
-            case Types.OTHER:
-            case Types.JAVA_OBJECT:
-            case Types.DISTINCT:
-            case Types.STRUCT:
-            case Types.ARRAY:
-            case Types.BLOB:
-            case Types.CLOB:
-            case Types.REF:
-            case Types.ROWID:
-            case Types.SQLXML:
-            case Types.DATALINK:*/
-
             // Unsupported (see above) types are treated as a text values.
```

Comment:
Is this comment still true when we are no longer returning ColumnType.TEXT in default Case?